docs/source/using/profiles.adoc

```diff
@@ -12,7 +12,23 @@ toc::[]
 == Overview
 
 {project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+Each {project} instance can be created with different configurations e.g memory, CPU, disk-size , add-ons etc and users can switch between different instances.
+Profile feature would help users to test different configurations without deleting an existing {project} instance.
+
+Each profile is completely independent of other profiles and things like cache, add-ons and persistent config are specific to only one profile.
+This means if you have set some specific configuration parameters for a profile, these settings does not apply to other profiles. 
+
+Currently we do not have any global config for profiles.
+So if you want to have a common configuration for all your profiles then you need to apply the settings for each profile.
+However you can use xref:../using/managing-minishif.adoc#environment-variables[environment variables] as a workaround to use same configuration settings for all profiles.
+
+
+Profile feature introduced a global --profile flag and few commands i.e. `profile list`, `profile set`, `profile delete`.
```

Comment:
--profile should be inside backticks `--profile`

Reply:
@anjannath yes, will fix it.